### PR TITLE
Post box

### PR DESCRIPTION
**Pull request by vangmayi98**
@flash29 PostBox complete for Sprint 1. Please review and merge it is everything looks good.

### CODE REVIEW

**Comment by flash29** [comment]
everything looks good merging the code.

File: frontend/src/dashboard/Home/Home.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ function Home() {
   return (
     <div className="App">
       <NavBar />
+      <PostBox />
+      This is the home page
```
Comment (flash29) [inline frontend/src/dashboard/Home/Home.js:28]: Delete this line "This is the home page"

File: frontend/package.json [hunk 2]
```diff
@@ -11,10 +11,19 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.25.0",
+    "body-parser": "^1.19.1",
+    "cors": "^2.8.5",
+    "express": "^4.17.2",
+    "mongoose": "^6.1.10",
```
Comment (flash29) [inline frontend/package.json:18]: are we using these libraries???
i dont think we are using axios, body-parser, express, mongoose, react-bootstrap, react-file-base64, nodemon.
if possible delete these unnecessary packages.

**Review by flash29 — COMMENTED** [review]
Please look into the comments

**Comment by vangmayi98** [comment]
@flash29 Deleted the comment on the homepage and the unnecessary packages from package-json.

**Comment by vangmayi98** [comment]
Added the page-auto-reload functionality.

**Comment by flash29** [comment]
everything looks good merging now

**Pull request merged**